spec/mongo/crypt/explicit_encryption_context_spec.rb

```diff
@@ -79,6 +78,40 @@
           context
         end.not_to raise_error
       end
+
+      context 'with verbose logging' do
+        before(:all) do
+          # Logging from libmongocrypt requires the C library to be built with the -DENABLE_TRACE=ON
+          # option; none of the pre-built packages on Evergreen have been built with logging enabled.
+          #
+          # It is still useful to be able to run these tests locally to confirm that logging is working
+          # while debugging any problems.
+          #
+          # For now, skip this test by default and revisit once we have determined how we want to
+          # package libmongocrypt with the Ruby driver.
+          skip "These tests require libmongocrypt to be built with the '-DENABLE_TRACE=ON' cmake option." +
+            " They also require the MONGOCRYPT_TRACE environment variable to be set to 'ON'."
+        end
```

Comment:
@p-mongo let me know if you're okay leaving the tests like this for now.

Reply:
I think it's fine, maybe link to the jira ticket for packaging if you are going to make other changes.